src/components/CaptureRoutes/CaptureRoutes.tsx

```diff
@@ -17,7 +17,7 @@ const CaptureRoutes = (): JSX.Element => {
     });
   }, [location, sendAnalytics]);
 
-  return <Outlet />;
+  return null;
```

Comment:
This component no longer wraps all the other components so doesn't need an outlet.